{
  try {
    this.extractor.getIntValue(this.bean);
    fail("Should have throw an exception");
  }
 catch (  final Exception e) {
  }
}
